v22.2/install-cockroachdb-mac.md

```diff
@@ -52,42 +52,65 @@ If you previously installed CockroachDB via Homebrew, run <code>brew uninstall c
 {{site.data.alerts.end}}
 </div>
 
+{% capture binary_arm_geos_unquarantine %}
+{% include_cached copy-clipboard.html %}
+~~~ shell
+xattr -d com.apple.quarantine lib/libgeos*
+~~~
+{% endcapture %}
+
+{% capture st_invalid_output %}
+~~~ shell
```

Comment:
Total nit: If this is output, do we want the `$` prompt / shell syntax highlight on this?

Reply:
No, because this is output from a query in the `cockroach` binary and there isn't anything specific to highlight.